test-site/about.html

```diff
@@ -4,6 +4,15 @@
     <meta charset="UTF-8">
     <title>About</title>
     <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0">
+    <script id="metrica" async src="http://stage.metrica.fun/metrica.js?tracking_id=00000148" ></script>
+    <script>
+        window._metricaTrackingConfig = window._metricaTrackingConfig || [];
```

Comment:
you are defining _metricaTrackingConfig as array, but above you work with it as with object.